File: dsa-course/how_to_solve/problem-2/ans.py
array = [1,2,4,5] 

sum = 11

# params 2
# return true/false

def findSum(arr, sum):
    for i in arr:
        for j in arr:
            if i+j == sum:
                return True
    return False

print(findSum(array, sum))


# 2nd approch

def findSum2(arr, sum) :
    storeDict = {}
    
    for i in arr:
        com = sum - i
        if com not in storeDict:
            storeDict[i] = True
        else:
            return True
    return False


print(findSum2(array, sum))

    

File: dsa-course/how_to_solve/problem-2/ans.js
const array = [1,3,4,5,6,2]

const sum = 9


const findSome = (arr, some) => {
    storeData = {}
    for (let i of arr){
        com = sum - i

        if (!storeData[com]) {
            storeData[i] = true
        }
        else{
            return  true
        }
    }
    return false
}

console.log(findSome(array, sum))
